composer.json

```diff
@@ -3,16 +3,17 @@
     "description": "PHP implementation of The Update Framework (TUF)",
     "type": "library",
     "require-dev": {
-        "phpunit/phpunit": "^8.5.8|^9",
+        "phpunit/phpunit": "^9",
```

Comment:
I'm assuming the only reason we'd need to support PHPUnit 8 is for PHP 7.2 support.